build.gradle

```diff
@@ -160,6 +160,7 @@ dependencies {
     // Include all jar-files in the 'lib' folder as dependencies
     implementation fileTree(dir: 'lib', includes: ['*.jar'])
 
+
```

Comment:
Does an AI bot create these empty lines?

I would recommend to review a commit using git gui!